scm-ui/ui-webapp/src/admin/plugins/containers/PluginsOverview.tsx

```diff
@@ -177,21 +177,28 @@ const PluginsOverview: FC<Props> = ({ installed }) => {
   const computeUpdateAllSize = () => {
     const outdatedPlugins = collection?._embedded?.plugins.filter((p: Plugin) => p._links.update).length;
     return t("plugins.outdatedPlugins", {
-      count: outdatedPlugins
+      count: outdatedPlugins,
     });
   };
 
   const renderPluginsList = () => {
+    let pluginCenterErrorNotification: React.ReactNode;
+    if (collection?.pluginCenterError) {
+      pluginCenterErrorNotification = <Notification type="danger">{t("plugins.pluginCenterError")}</Notification>;
```

Comment:
```suggestion
      pluginCenterErrorNotification = <Notification type="warning">{t("plugins.pluginCenterError")}</Notification>;
```

How about warning instead of danger?